### ISSUE TITLE
Clusterrolebinding is not created by ./test/e2e-tests.sh, but removed by ./test/e2e-tests.sh

### ISSUE THREAD

**Issue by houshengbo** [knative/test-infra#894]
The clusterrolebinding should be created before running the command ./test/e2e-tests.sh --run-tests, which means it is out of the scope of e2e-tests.sh to create this clusterrolebinding.
However, in the end of ./test/e2e-tests.sh, the clusterrolebinding is removed. The behavior is not symmetric. Suggest not to remove the clusterrolebinding in ./test/e2e-tests.sh.

**Comment by adrcunha** [comment]
It is created for the cluster if it doesn't exist

https://github.com/knative/test-infra/blob/80d9eccb2177e903a89c3aedceaf0d20fd4d1e0c/scripts/e2e-tests.sh#L298

It's not removed by the script above. But for serving, it's deleted (not failing if not found though):

https://github.com/knative/serving/blob/master/test/e2e-common.sh#L228

**Comment by adrcunha** [comment]
Indeed, the binding should only be removed if it was created by the test. Since the creation is not being tracked, deleting it might cause an inconsistent/unexpected state in already existing clusters.

Moving this to knative/serving.
